includes/functions.php

```diff
@@ -167,3 +168,19 @@ function get_stories_order() {
 		]
 	);
 }
+
+/**
+ * Renders lightbox markup.
+ *
+ * @return void
+ */
+function render_stories_lightboxes() {
+
+	if ( is_amp() ) {
+		Renderer::render_stories_with_lightbox_amp();
+	} else {
+		Renderer::render_stories_with_lightbox_noamp();
+	}
+}
+
+add_action( 'wp_footer', 'Google\Web_Stories\render_stories_lightboxes' );
```

Comment:
Could you elaborate a bit on the reason for this change around `render_stories_lightboxes` and making `render_stories_with_lightbox_noamp`/`render_stories_with_lightbox_amp` static?

This doesn't seem like a very developer-friendly API as it makes it hard to grasp what is actually going on.

* `render_stories_lightboxes` is a function for seemingly no reason.
* `render_stories_lightboxes` calls two static methods which rely on the `Renderer` already having generated some output.
* There is no context on why this is being done outside `Renderer`

cc @spacedmonkey WDYT?

Reply:
I've have questions for this as well, so while I was adding style changes for the Core Themes, I found that most issues are relating to the `z-index` or stacking context. 
Since we were rendering `lightboxes` in the `web-stories-list` wrapper, sometimes other elements were showing on top of the stories lightbox. For example for `twenty-fourteen`, I kept the stories in sidebar and enabled the customizer stories, the lightbox for the sidebar stories has header stories on top of it.

This change mitigates that issue as we move our lightbox into top most context of the page. But I didn't know where to put this piece of code, need your suggestion.

Reply:
What is this doing and why is it rendered on every page? it seems super wasteful to render it everywhere, even if it is not needed.

Reply:
Functions are designed to part of the public api, this should live in a class and should be conditionally loaded.